src/Explorer/Controls/NotebookGallery/__snapshots__/GalleryViewerComponent.test.tsx.snap

```diff
@@ -8,95 +8,6 @@ exports[`GalleryViewerComponent renders 1`] = `
     onLinkClick={[Function]}
     selectedKey="OfficialSamples"
   >
-    <PivotItem
```

Comment:
Does this mean we're no longer testing the public gallery because it's behind a feature flag? If so, is there a way to set the feature flag for the test, or is this not worth testing?

Reply:
these snapshots are testing only the load state of the component. We dont have the whole component available now dude to the feature flags, so this is fine.